spec/models/callback_modifier_spec.rb

```diff
@@ -29,7 +29,7 @@
           it 'should default to after destroy' do
```

Comment:
Not sure how I missed that while pulling in the cherry-picked commit... Fixed in 65c3ce4426d2da1fa51df8b3e38091cc6def685a